wiki/Daily_challenge/en.md

```diff
@@ -0,0 +1,65 @@
+# Daily Challenge
+
+The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a *daily* streak by passing beatmaps consecutively with increasing difficulty.
+
+Each day will introduce one new map from the [featured artist](/wiki/People/Featured_Artists) listing with increased difficulty. Regularly after one week however, the difficulty would be reset again.
+
+Sometimes, the maps could come with a mod forced onto users, which required them to pass the map with the selected mod.
+
+## Game menu
+
+*Notice: Daily challenge requires an [osu! account](/wiki/Registration), and is not available for [silenced](/wiki/Silence) players.*
+
+From the main menu, the daily challenge menu can be accessed with the following steps:
+
+1. Click the `Play` button or press `P`.
+2. Click the `Daily challenge` button or press `D`.
+
+![](img/Daily-challenge-menu.png "Screenshot of the daily challenge menu as of November 7, 2024.") 
+
+On entering, the user is presented with an intro showing the beatmap the user has to pass, and the mods they have to pass it with.
+After the intro ends, the user can view the data for today's challenge. The left side displays various score-related information like the total pass count and the cumulative total score. In the middle, a leaderboard shows the top scores achieved by players. Users are able to discuss the daily challenge in the chat on the right.
+
+## Beatmap difficulty range
+
+
+
+![](img/Beatmap-selection-criteria.png "Specific star ratings beatmaps have to follow for the entire week")
+
+
+
+## Streak milestones
+
+From playing the daily challenge, you can reach different milestones that will affect the rank of the number representing your streaks in your user profile.
+
+<!---  #### Rank breakdown 
+
+Note:  I didn't see the Total and Week requirements in the reference or i didn't understand the code, so some help with this would be nice
+
+| Rank  | Total streak | Daily streak | Weekly streak |
```

Comment:
Yea, i guessed so, but I don't know how to do that so I didn't do it 
(also I don't have the values for Total and Weekly streaks yet, and I think that's more important if we're listing this)

I'll look though the contributing guides to check on how to do it

Reply:
Im thinking that instead of making the numbers the rank colour (i feel this would get tacky) we could make a 
"badge "  but users wouldn't be able to get them and they would be basically only a background with the rank name in its colour 
(i feel like this would be the simplest and the best way to do it for now, unless you have a better idea)
(after some time we can give them better badges and reward users with the higher ones once they reach the desired rank)